docs/content/operations/dashboard.md

```diff
@@ -106,27 +89,46 @@ rule = "Host(`traefik.example.com`) && PathPrefix(`/api`, `/dashboard`)"
 
 ## Insecure Mode
 
-This mode is not recommended because it does not allow the use of security features.
+When _insecure_ mode is enabled, one can access the dashboard on the `traefik` port (default: `8080`) of the Traefik instance,
+at the following URL: `http://<Traefik IP>:8080/dashboard/` (trailing slash is mandatory).
+
+This mode is **not** recommended because it does not allow security features. For example, it is not possible to add an authentication middleware with this mode.
```

Comment:
```suggestion
This mode is **not** recommended because it does not allow security features.
For example, it is not possible to add an authentication middleware with this mode.
```